test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -68,39 +78,88 @@ def test_delete_schedulers(self):
         """
         Delete a scheduler object succcessfully .
         """
-        a = {'sched_cycle_length': 12000}
-        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
-        rc = self.server.manager(MGR_CMD_DELETE, SCHED, a, id="test1")
-        self.assertTrue(rc == 0)
+        try:
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="TestCommonSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
 
     def test_duplicate_scheduler_name(self):
         """
         Check for the scheduler object name.
         """
-        a = {'sched_cycle_length': 12000}
-        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
         try:
-            rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
+            self.server.manager(MGR_CMD_CREATE,
+                                SCHED,
+                                id="TestCommonSched")
         except PbsManagerError, e:
-            self.assertTrue(
-                'qmgr: Error (15211) returned from server' in e.msg[1])
+            if self.server.get_op_mode() == PTL_CLI:
+                self.assertTrue(
+                    'qmgr: Error (15211) returned from server' in e.msg[1])
```

Comment:
This test will pass in API mode.  Same with similar try/except blocks throughout the test.